tests/unit/plugins/inventory/test_proxmox.py

```diff
@@ -12,6 +12,7 @@
 from ansible.errors import AnsibleError, AnsibleParserError
 from ansible.inventory.data import InventoryData
 from ansible_collections.community.general.plugins.inventory.proxmox import InventoryModule
+from ansible_collections.community.general.plugins.modules.cloud.misc.proxmox_kvm import parse_dev, parse_mac
```

Comment:
These unit tests only run when changes are made to the `proxmox` inventory plugin.
Copy the relevant units and imports to a new file `test_proxmox_kvm.py` [here](https://github.com/ansible-collections/community.general/tree/main/tests/unit/plugins/modules/cloud/misc) to have them run as intended. (And revert this file.)

Reply:
Thanks for spotting that, I completely miss that...